gems/quilt_rails/README.md

```diff
@@ -215,18 +217,205 @@ class ReactController < ApplicationController
 end
 ```
 
+### Performance
+
+#### Reportable
+
+The `Quilt::Performance::Reportable` mixin is intended to be used in Rails controllers, and provides only the `process_report` method. This method handles parsing an incoming report from [@shopify/react-performance's](https://www.npmjs.com/package/@shopify/react-performance) `<PerformanceReport />` component (or a custom report in the same format) and sending it to your application's StatsD endpoint as `distribution`s using [`StatsD-Instrument`](https://rubygems.org/gems/statsd-instrument).
+
+> **Note** `Quilt::Performance::Reportable` is usable even if you are using purely client-side React through `sewing_kit_script_tag`.
+
+```ruby
+class PerformanceController < ApplicationController
+  include Quilt::Performance::Reportable
+
+  def index
+    process_report
```

Comment:
Waffled on this name a lot, what this does in full is:
- parse the `params` sent to the `PerformanceReportController` into a `Report` instance
- call `Quilt::Performance::Client.send!` on the constructed instance

So maybe like `parse_and_send_report` but that sounds weird :/